scripts/meetingbaas_api.py

```
@@ -195,9 +246,10 @@ def create_meeting_bot(
 
         response = requests.post(url, json=config, headers=headers)
 
-        if response.status_code == 200:
+        if response.status_code == 201:
             data = response.json()
-            bot_id = data.get("bot_id")
+            # v2 response: {"success": true, "data": {"bot_id": "..."}}
+            bot_id = data.get("data", {}).get("bot_id")
             logger.info(f"Bot created with ID: {bot_id}")
```

Ruff (0.15.11)

[warning] Logging statement uses f-string

(G004)